streams/in.m3u

```diff
@@ -11,7 +11,8 @@ https://feeds.intoday.in/aajtak/api/aajtakhd/master.m3u8
 https://aajtaklive-amd.akamaized.net/hls/live/2014416/aajtak/aajtaklive/live_404p/chunks.m3u8
 #EXTINF:-1 tvg-id="AajTak.in",Aaj Tak (360p) [Geo-blocked]
 https://lmil.live-s.cdn.bitgravity.com/cdn-live/_definst_/lmil/live/aajtak_app.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="Aastha.in",Aastha (480p)
+#EXTINF:-1 tvg-id="A JattTube.in",A JattTube
+http://tiny.cc/jattvlink
```

Comment:
```suggestion
#EXTINF:-1 tvg-id="SwagJatt.in",SwagJatt (720p)
```
Or

```suggestion
#EXTINF:-1 tvg-id="JattTubeCom.in",JattTubeCom (720p)
```

And

```suggestion
https://ssh101.bozztv.com/ssh101/swagjatt/playlist.m3u8
```

Reply:
As the channel isn't in the database, no tvg-id could correspond.